radixdlt/src/main/java/com/radixdlt/counters/SystemCounters.java

```diff
@@ -32,6 +32,7 @@
 		LEDGER_STORED,
 		MEMPOOL_COUNT,
 		MEMPOOL_MAXCOUNT,
+		MESSAGES_INBOUND_BADSIGNATURE,
```

Comment:
Added signature checking for SignedMessages (oops) and counter for bad signatures.